<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Camera_View">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>551</width>
      <height>441</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Enumerate">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>470</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Find Device</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Open">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>20</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Close">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>20</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="Acquisition">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>70</y>
      <width>241</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Frame Acquisition</string>
    </property>
    <widget class="QCheckBox" name="Continuous">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>111</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Continuous</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="Trigger">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>91</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Trigger</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Time_delay">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>90</y>
       <width>91</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Trigger Delay (ÂµS)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Start">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>111</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Start Grabbing</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Stop">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>120</y>
       <width>111</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop Grabbing</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SoftwareTrigger">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Trigger</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="Paramater">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>240</y>
      <width>241</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Device Paramters</string>
    </property>
    <widget class="QLineEdit" name="FrameRate">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>91</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Frame Rate</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Expotime">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>60</y>
       <width>91</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Exposure Time</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Gain</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Gain">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>90</y>
       <width>91</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="SetParam">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>120</y>
       <width>101</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Set</string>
     </property>
    </widget>
    <widget class="QPushButton" name="GetParam">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>101</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Get</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>400</y>
      <width>241</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Save Frame</string>
    </property>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Height</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Height">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>91</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Width</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Width">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>50</y>
       <width>91</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Capture">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>80</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Capture</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
